src/hooks/useInactiveListener.ts

```diff
@@ -0,0 +1,40 @@
+import { useWeb3React as useWeb3ReactCore } from '@web3-react/core'
+import { useEffect } from 'react'
+import { injected } from '../connectors'
+
+export default function useInactiveListener(suppress = false) {
+  const { active, error, activate } = useWeb3ReactCore() // specifically using useWeb3React because of what this hook does
+
+  useEffect(() => {
+    const { ethereum } = window
```

Comment:
Is working on the global `ethereum` object absolutely necessary? We want to avoid globals if we can.

Reply:
It's necessary for only inactive listener